packages/react-components/src/Popup.tsx

```diff
@@ -2,6 +2,30 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Popup } from 'semantic-ui-react';
+import { Popup as PopupBase, StrictPopupProps } from 'semantic-ui-react';
+import React from 'react';
+import styled from 'styled-components';
 
-export default Popup;
+function Popup({ content, trigger, className }: StrictPopupProps): React.ReactElement {
+    return <PopupBase
+      content={content}
+      trigger={trigger}
+      className={className}
+    />
+}
+
+export default styled(Popup)`
+    background-color: #1B1C1D !important;
+    ::before {
+        background-color: #1B1C1D !important;
+    }
+
+    .ui--popUpText, p {
+        color: #FFFFFF;
+        font-size: 0.9em;
+        display: flex;
+        > i {
```

Comment:
In old-school css styles, css styles would not be suggested to be applied directly on html elements like `p`, `a`, etc as well, which could always be achieved by applying css classname on those elements. 

In `cennznet-ui`, all application basic element styles are defined in [react-components/src/styles/index.ts](https://github.com/cennznet/ui/blob/develop/packages/react-components/src/styles/index.ts).

In this case, it is fine, as it is scoped inside a component, it seems much more easier in this way.